### ISSUE TITLE
Import paths extracted from dictionaries and lists, not resolved

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-jsonpreprocessor#382]
JSON:

```
"import_files" : {
                   "fileAA" : "./dynamic_imports/AA/imported.AA.jsonp",
                   "fileBB" : "./dynamic_imports/AA/BB/imported.BB.jsonp",
                   "fileCC" : "./dynamic_imports/AA/BB/CC/imported.CC.jsonp"
                 },
"[import]" : ${import_files}['fileAA']
```

Result:

`Error: 'File '%import_files%['fileAA']' is not existing!'!`

Alternative:

`"[import]" : "${import_files}['fileAA']"`

Result:

`Error: 'File '%import_files%['fileAA']__ConvertParameterToString__' is not existing!'!`

The same with import paths from lists.

Expected: Paths resolved and files imported.

**Comment by HolQue** [comment]
JPP_1109
JPP_1110

**Comment by HolQue** [comment]
Alternatively both ways (with and without quotes) executed together:

```
"[import]" : ${import_files}['fileAA'],
"[import]" : "${import_files}['fileBB']"
```

Result:

`'File '%import_files%['fileBB']__ConvertParameterToString__' is not existing!'!`

This is related to the '`fileBB`' error. The '`fileAA`' error is not shown.

Expected: Already the first error ('`fileAA`') should stop the computation.

**Comment by namsonx** [comment]
Hello Holger,

This issue was solved on stabi branch. Could you please help me verify it again.

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
merged for version 0.13.1
